qiskit/ignis/experiments/calibration/methods/cross_resonance.py

```diff
@@ -57,29 +57,43 @@ def __init__(self,
         # something like qubit property table where f01, anharmonicity, T1, T2, etc... exist.
         freq01 = None
 
+        self._name = 'cr_amp'
+        self._qubits = qubits  # todo move to base class
+        self._inst_def = inst_def  # todo move to base class
+        self._calibration_group = calibration_group
+
         u_ch = inst_def.get_channel(qubits, ControlChannel)
 
-        # Create a list of amp parameters on the control channel.
+        # Create a list of amp parameters on the control channel and default to global.
         u_ch_inst = inst_def.get_schedule(cr_name, qubits).filter(channels=[u_ch]).instructions
```

Comment:
this needs to be bit more careful because frame change can be inserted. It's more safe to filter instruction type too.

Reply:
Done in 3852844